### DISCUSSION TITLE
Getting thumbnails from Wikidata

### DISCUSSION THREAD

**Discussion by kbseah** [theosanderson/taxonium#578] [Q&A]
Hello Theo, and thanks for this great tool. I've been using Taxonium at work to quickly visualize trees and appreciate how it works out of the box without much fuss.

I'm curious about how you set up the "Exploring NCBI Taxonomy & Wikipedia" example tree, in particular how you get the image links and information from Wikidata. I'm guessing there is some kind of custom config you wrote for this? I'm asking because I'd like to build on this to link to related media files on Wikimedia Commons. Perhaps you've already had some thoughts in this direction.

Thanks!

**Comment by theosanderson** [comment]
Hi @kbseah ,

I just wanted to sincerely apologise having finally noticed I never replied to this. I realise this answer will be much to late for you and am sorry for that. But for anyone reading in future: there is currently a harcoded metadata field called `ThumbnailURL` that achieves this.

https://github.com/search?q=repo%3Atheosanderson%2Ftaxonium%20thumbnailurl&type=code

So the JSONL used for the example you cite contains items like:

`{"name":"35129","x_dist":206.25,"y":9946.0,"mutations":[],"meta_name":"Asterionella","meta_rank":"genus","meta_WikiPage":"[Asterionella](https://en.wikipedia.org/wiki/Asterionella)","meta_AlsoKnownAs":"","meta_ThumbnailURL":"https://upload.wikimedia.org/wikipedia/commons/thumb/2/29/Asterionella_formosa.jpg/200px-Asterionella_formosa.jpg","meta_LatinName":"Asterionella","parent_id":14068,"node_id":14069,"num_tips":1}`

Thank you for your question and sorry that it did not receive a timely answer.

All the best

Theo

**Reply by kbseah** [reply]
Thanks for getting back to me, still good to know!
